WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -1086,6 +1091,30 @@ import WordPressComAnalytics
         }
     }
 
+    func backgroundFetch(_ completionHandler: ((UIBackgroundFetchResult) -> Void)?) {
+        print("=========>>>>>>>>>>> backgroundFetch")
+        print("=========>>>>>>>>>>> \(String(describing: readerTopic?.title))")
+
+        let lastSeenPostID = (tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID
+
+        syncHelper.backgroundSync(success: { [weak self, weak lastSeenPostID] in
+            let newestFetchedPostID = (self?.tableViewHandler.resultsController.fetchedObjects?.first as? ReaderPost)?.postID
```

Comment:
I always have to execute the completionHandler so exiting early is not a good idea here.